EntranceShuffle.py

```diff
@@ -91,20 +91,24 @@ def connect_fairy(world, entrancename, exitname):
 
 # these are connections that cannot be shuffled and always exist.
 # They logically separate areas that do not cross loading zones
-mandatory_connections = [('South Mailbox', 'Mailbox'),
+mandatory_connections = [ ('Cursed Underground', 'Clock Tower'),
+                                                ('South Mailbox', 'Mailbox'),
                                                 ('East Mailbox', 'Mailbox'),
                                                 ('North Mailbox', 'Mailbox'),
```

Comment:
Aren't these `South/East/North` mailboxes superfluous? Can't it just be `South/East/North Clock Town` that all connect to `Mailbox`?